test/smoke_test.rb

```diff
@@ -33,6 +35,50 @@ def test_it_finds_all_movies
     assert_equal Movie.all.size, search.result.size
   end
 
+  def test_scope_filter
+    search = MovieSearch.new("search" => {})
+    assert_equal 40, search.result.size
+    assert search.filter(:classic).facet.include? FortyFacets::FacetValue.new(:classic, 6, false)
+    assert search.filter(:classic).facet.include? FortyFacets::FacetValue.new(:non_classic, 34, false)
+
+    search = MovieSearch.new("search" => { "classic" => "classic" })
+    assert_equal 6, search.result.size
+    assert search.filter(:classic).facet.include? FortyFacets::FacetValue.new(:classic, 6, true)
+    assert search.filter(:classic).facet.include? FortyFacets::FacetValue.new(:non_classic, 34, false)
+
+    search = MovieSearch.new("search" => { "classic" => "non_classic" })
+    assert_equal 34, search.result.size
+    assert search.filter(:classic).facet.include? FortyFacets::FacetValue.new(:classic, 6, false)
+    assert search.filter(:classic).facet.include? FortyFacets::FacetValue.new(:non_classic, 34, true)
+
+    search = MovieSearch.new("search" => { "classic" => ["non_classic", "classic"] })
+    assert_equal 40, search.result.size
+    assert search.filter(:classic).facet.include? FortyFacets::FacetValue.new(:classic, 6, true)
+    assert search.filter(:classic).facet.include? FortyFacets::FacetValue.new(:non_classic, 34, true)
+  end
+
+  def test_scope_filter_without_path
```

Comment:
Assignment Branch Condition size for test_scope_filter_without_path is too high. [56.14/15]